### DISCUSSION TITLE
Big Sur OTA doesn't work

### DISCUSSION THREAD

**Discussion by rajibxo** [daliansky/XiaoMi-Pro#592] [Q&A] [answered]
system- KBL
OC- v1.6.1
Big Sur doesn't show new updates, I updated from 11.2 to 11.5 by just installing whole 12 GB Mac OS Big Sur! even most of the application doesn't update...whats the issue here...?? btw its from previous refi versions... 

also on the other note, does anyone know the solution to Mi notebook pro rattling fan noise issue?

**Comment by stevezz3** [comment, marked as answer]
@rajibxo OTA updates should work on v1.6.2
For applications not show updates, that's normal even on real Macs.
For fan noise, I recommend you update to the latest BIOS version. See https://github.com/daliansky/XiaoMi-Pro-Hackintosh/tree/main/BIOS

**Reply by rajibxo** [reply]
OTA is fixed, seems to be some unwanted files in EFI partition causing the issue. My BIOS is 906, the A07 or E07 which one is the new, are these 3rd party? I got rid of windows...no way of running. exe maybe...

Thanks.

**Reply by stevezz3** [reply]
Depends on your model. If you are TM1701, then the latest one is XMAKB5R0P0E07. All BIOS packets are from Xiaomi official as claimed in [Introduction](https://github.com/daliansky/XiaoMi-Pro-Hackintosh/tree/main/BIOS#introduction) and [Credits](https://github.com/daliansky/XiaoMi-Pro-Hackintosh/tree/main/BIOS#credits).

**Reply by rajibxo** [reply]
got it...yes mine is TM1701, thanks a lot
